src/js/components/PropertyList/PropertyList.js

```diff
@@ -0,0 +1,45 @@
+import React, { forwardRef, useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Grid } from '../Grid';
+import { ResponsiveContext } from '../../contexts/ResponsiveContext';
+import { PropertyListContext } from './PropertyListContext';
+
+const PropertyList = forwardRef(
+  (
+    {
+      a11yTitle,
+      align,
+      columns: columnsProp,
+      direction = { list: 'column', property: 'row' },
```

Comment:
would this make more sense to be 
` direction = { name: 'column', value: 'row' },`

then we would have to change the names below as well
just a little taken back there's `list` and `property` but then `nameProp` and `valueProps`

Reply:
The direction prop is regarding the overall `list` vs the direction of individual properties, so the `property` key determines direction of name & value.